docs/contributing/index.rst

```diff
@@ -3,59 +3,33 @@
 Contributing to Weblate
 =======================
 
+Language
+--------
 
-There are dozens of ways to contribute in Weblate. Any help is welcomed, be it
-coding, graphics design, documentation or sponsorship:
+`Translate <https://hosted.weblate.org/projects/weblate/>`_ Weblate itself,
+and point out errors in source strings while you do. Suggest source string changes, or
+click "Source string location" in the lower right of the regular editor to have
+a go yourself.
 
-* :doc:`issues`
-* :doc:`start`
-* :ref:`translating-weblate`
-* :ref:`contributing-docs`
-* :ref:`funding-weblate`
+Here in the documentation, click :guilabel:`Edit on GitHub` atop the page to do the same thing.
+`Open an issue https://github.com/WeblateOrg/weblate/issues/new?assignees=comradekingu&labels=ux&template=bug_report.md>`_
+if you know something is wrong or have an issue with the documentation, but not how to fix it.
+Remove the template stuff; a link and description will do.
 
-.. _translating-weblate:
+Graphics
+--------
+The visuals are in https://github.com/WeblateOrg/graphics
 
-Translating Weblate
--------------------
+`Simple Icons issue https://github.com/simple-icons/simple-icons/issues/5085>`_
+`Fork Awesome issue https://github.com/ForkAwesome/Fork-Awesome/issues/218#issuecomment-611892888>`_
 
-Weblate is being `translated <https://hosted.weblate.org/>`_ using Weblate itself, feel
-free to take part in the effort of making Weblate available in as many human languages
-as possible.
+Funding
+-------
 
-.. _contributing-docs:
-
-Contribute to Weblate documentation
------------------------------------
```

Comment:
We definitely need a documentation how to contribute to the documentation. Several people have asked us for a guidance here and this is the initial version of that. The motivation is to make it both easy to contribute and easy to review and merge. Huge pull requests which take weeks to review are not good for either side - the contributor feels bad because there is need for continuous rewriting and reviewers get lost in what has part has been approved and which not.

Reply:
@nijel I will grant you both it needs doing (with some level of urgency below that of fixing the access docs being a mess and appearing first for translation (without being marked up with terminology or contextualised), _and_ that people ask about it.

Instead of something that makes it easy to contribute, you have made something that I am walking away from. The rules in this are moronic and reactionary. I value your time, and I wish you the best of luck.

As of this year I can actually call myself a professional libre software documentation writer. It isn't lucrative, but I mention it so that with _every_ bit of experience and conviction I have, I can say you have the wrong ideas, and end up in downfall of the commons on every level. Fat-Zer has qualities that I don't. I hope you can make him the head of the documentation. I can offer to pay him to do so.

https://github.com/WeblateOrg/weblate/pull/5566#issuecomment-801842143
Here you have an example of the disconnect between being asked how to do something, and the idea that it makes one capable of answering.